lms/djangoapps/verify_student/models.py

```diff
@@ -120,26 +120,6 @@ def expiration_datetime(self):
         days_good_for = settings.VERIFY_STUDENT["DAYS_GOOD_FOR"]
         return self.created_at + timedelta(days=days_good_for)
 
-    def should_display_status_to_user(self):
-        """Whether or not the status from this attempt should be displayed to the user."""
-        raise NotImplementedError
-
-    def active_at_datetime(self, deadline):
-        """Check whether the verification was active at a particular datetime.
-
-        Arguments:
-            deadline (datetime): The date at which the verification was active
-                (created before and expiration datetime is after today).
-
-        Returns:
-            bool
-
-        """
-        return (
-            self.created_at < deadline and
-            self.expiration_datetime > datetime.now(pytz.UTC)
-        )
-
 
 class IDVerificationAggregate(IDVerificationAttempt):
```

Comment:
I've kept this class here as the migration introducing this table was a dependency of the other migration.